nautobot/docs/models/dcim/cable.md

```diff
@@ -32,3 +32,12 @@ Traced from Interface 1 on Device A, Nautobot will show the following path:
 * Cable 1: Interface 1 to Front Port 1
 * Cable 2: Rear Port 1 to Rear Port 2
 * Cable 3: Front Port 2 to Interface 2
+
+A cable can also be traced through a circuit.  Due to the path caching, the Circuit should have both terminations added before connecting a cable.  If a cable is already connected, changing the cable status from `Connected` to another status such as `Planned` followed by changing the status back to `Connected` will trigger a path refresh. 
```

Comment:
Are we absolutely certain we can't detect this event using a signal and cause this to happen automatically? This seems like an undesirable user experience. Failing that, could we implement a new button or explicit UI/API action to "rebuild paths" or something like that?

Reply:
We should create a ticket to implement a button or something for rebuild paths.  As far as the event/signal, I can try to explain how this happens.

Reply:
Since cable terminations are also path endpoints, the following is valid [Interface] -- Cable -- [Circuit Termination]. So, a cable path is created with Interface as the origin and Circuit Termination as the destination. If you then add [Circuit Termination 2] and [Interface 2] and connect them with a Cable, refresh_paths function will look for any paths that include [Interface 2] and [Circuit Termination 2].  Since the first path was valid and only included [Interface] and [Circuit Termination] it will not be found by the refresh_paths query and called for update.

We might be able to write a new signal that could detect this.

We could change Circuit Terminations to not be a Path Endpoint, but that will break other use cases for the Cable Termination.